test/admin-hpa/features/support/env.rb

```diff
@@ -61,7 +61,6 @@
   config.exact = true
   config.ignore_hidden_elements = false
   config.visible_text_only = true
-  config.default_max_wait_time = 7
```

Comment:
when its not clear what unit the config item requires we create a variable that describes it, and assign that to the property.
Like this...
```
sevenSeconds = 7
config.default_max_wait_time = sevenSeconds
```

Reply:
Will do going forward, this config was removed as part of this commit